src/hooks/useIsDesktop.hook.tsx

```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from 'react';
+
+const useIsDesktop = (breakpoint: number = 1440) => {
```

Comment:
its needs to be constant breakpoint not a parameter. you can define the breakpoints with enum